tests/components/kodi/__init__.py

```diff
@@ -30,12 +28,8 @@ async def init_integration(hass) -> MockConfigEntry:
     entry = MockConfigEntry(domain=DOMAIN, data=entry_data, title="name")
     entry.add_to_hass(hass)
 
-    with patch("homeassistant.components.kodi.Kodi.ping", return_value=True), patch(
-        "homeassistant.components.kodi.Kodi.get_application_properties",
-        return_value={"version": {"major": 1, "minor": 1}},
-    ), patch(
-        "homeassistant.components.kodi.get_kodi_connection",
-        return_value=MockConnection(),
+    with patch(
```

Comment:
It's better to patch the library rather than the integration code

Reply:
I also extended the ìnit_integration` with the latest commits.